packages/client-adapters/dist/alpheios-client-adapters.node.js

```diff
@@ -5629,7 +5629,7 @@ class AlpheiosLexiconsAdapter extends _clAdapters_adapters_base_adapter__WEBPACK
           lastAlt.push(strippedAll[0])
         }
       }
-      if (lastAlt.length > 0) {
+      if (data && lastAlt.length > 0) {

```

Comment:
Added for the cases when data is uploaded from remote source too long, and user has time to request a new query